src/wire/udp.rs

```diff
@@ -230,25 +231,26 @@ impl<'a> Repr<'a> {
         Ok(Repr {
             src_port: packet.src_port(),
             dst_port: packet.dst_port(),
-            payload:  packet.payload()
         })
     }
 
     /// Return the length of a packet that will be emitted from this high-level representation.
```

Comment:
Is this comment still accurate? I imagine a higher level representation would actually emit the payload as well?

Reply:
Nice catch, fixed!